MoviesApp/MosaicCell.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  MosaicCell.swift
+//  MoviesApp
+//
+//  Created by Dubrouski Nikita on 21.07.23.
+//
+
+import UIKit
+
+class MosaicCell: UICollectionViewCell {
+    
+    static let identifer = "kMosaicCollectionViewCell"
+
+    var imageView = UIImageView()
+    var assetIdentifier: String?
+    var title: String = ""
+    var poster: String = ""
+
+    
```

Comment:
remove whitespace

Reply:
extension for id

extension UIView {
static var id: String { String(describing: self) } 
}

create stored properties instead of setup in lifecycle